### PR TITLE
added Programming in C Section

### PR DESCRIPTION
**Pull request by kunalp78**
Pls, Add my contribution and me as well. I contributed to the Section of programming in C resources😘 @aneagoie

### CODE REVIEW

File: ProgrammingInC.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- [Microcontroller Embedded C Programming: absolute beginners](https://www.udemy.com/course/microcontroller-embedded-c-programming/)
+
+- [Fork CPP](https://www.geeksforgeeks.org/fork-cpp-course-structure) <h1>A good course for beginners</h1>
+
+- [EDU](https://codeforces.com/edu/course/2) Advanced course
+
+- [C++ For Programmers](https://www.udacity.com/course/c-for-programmers--ud210) Learn features and constructs for C++
+
+### Free Mentor
```
Comment (arjunkharbanda) [inline ProgrammingInC.md:41]: Remove this  no-self promotion is allowed

File: ProgrammingInC.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Videos
+
+- [Basic C Programming](https://www.youtube.com/watch?v=rLf3jnHxSmU&list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
+
+- [C Programming & Data Structures](https://www.youtube.com/watch?v=4OGMB4Fhh50&list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM)
+
+- [C Programming Edureka](https://www.youtube.com/watch?v=8PopR3x-VMY)
+
+## Git Books
```
Comment (arjunkharbanda) [inline ProgrammingInC.md:13]: organize it better 
list all the books under a single heading

**Review by arjunkharbanda — CHANGES_REQUESTED** [review]

File: ProgrammingInC.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+**[Back](/README.md/)**
+
+# Programming in C
```
Comment (arjunkharbanda) [inline ProgrammingInC.md:3]: Rename the file C Programming.md and also add the link to README.md

**Review by arjunkharbanda — CHANGES_REQUESTED** [review]

File: ProgrammingInC.md (hunk 1 above)
Comment (kunalp78) [reply ProgrammingInC.md]: I did it @arjunkharbanda

File: ProgrammingInC.md (hunk 2 above)
Comment (kunalp78) [reply ProgrammingInC.md:13]: brother that's how I want to represent. there is no restriction on my way...its said by the owner himself(@aneagoie )

**Pull request merged**